python/delta_sharing/rest_client.py

```diff
@@ -209,21 +211,39 @@ def __auth_basic(self, profile):
             }
         )
 
+    def set_sharing_capabilities_header(self):
+        delta_sharing_capabilities = (
+            DataSharingRestClient.DELTA_AND_PARQUET_RESPONSE_FORMAT + ';' +
+            DataSharingRestClient.DELTA_READER_FEATURES
+        )
+        self._session.headers.update(
+            {
+                DataSharingRestClient.CAPABILITIES_HEADER: delta_sharing_capabilities,
+            }
+        )
+
+    def remove_sharing_capabilities_header(self):
+        self._session.headers.update(
```

Comment:
are we able to remove the key from the headers?

Reply:
you mean the authorization bearer token?

`def __auth_bearer_token(self, profile):
        self._session.headers.update(
            {
                "Authorization": f"Bearer {profile.bearer_token}",
                "User-Agent": DataSharingRestClient.USER_AGENT,
            }
        )`

Reply:
fixed